service-workers/service-worker/clients-matchall-order.https.html

```diff
@@ -142,8 +144,8 @@
   }).then(_ => {
     frameResultList.forEach(result => result.top.remove());
     extraWindowResult.top.remove();
-  }).then(_ => {
-    return service_worker_unregister_and_done(t, opts.scope);
+
+    t.done();
```

Comment:
is there actually a need to call t.done()? It always seems suspicious to me if a promise_test calls done(), since the promise resolving itself should have that same effect?

Reply:
You're right: that's superfluous.